examples/log-ingestion/docker-compose-dataprepper.yaml

```
@@ -0,0 +1,15 @@
+version: ‘3.7’
+services:
+  data-prepper:
+    image: opensearchproject/data-prepper
```

Let's use the major version here so that it continues to run even with breaking changes.

```suggestion
    image: opensearchproject/data-prepper:2
```